api/apps/api/src/modules/projects/block-guard/marxan-block-guard.service.ts

```diff
@@ -46,5 +57,53 @@ export class MarxanBlockGuard implements BlockGuard {
       throw new BadRequestException(
         `Project ${projectId} editing is blocked because of pending marxan run`,
       );
+
+    //TODO check if project has pending imports
+  }
+
+  async ensureThatScenarioIsNotBlocked(scenarioId: string): Promise<void> {
```

Comment:
minor: I suggest to lightly refactor `ensureThatProjectIsNotBlocked()` above so that both functions use similar steps/checks